### PR TITLE
[ci][build] fix commit id

### PR DESCRIPTION
**Pull request by youkaichao**
try to fix commit id

### CODE REVIEW

**Comment by DarkLight1337** [comment]
Let's see if the tests here pass. If so, we can merge it instead of #6413

File: .buildkite/test-pipeline.yaml [hunk 1]
```diff
@@ -194,7 +194,9 @@ steps:
   command: pytest -v -s test_logits_processor.py
 
 - label: Utils Test
-  command: pytest -v -s test_utils.py
+  commands:
+    - pytest -v -s test_utils.py
+    - pytest -v -s tests/test_embedded_commit.py
```
Comment (DarkLight1337) [inline .buildkite/test-pipeline.yaml:199]:
```suggestion
    - pytest -v -s test_embedded_commit.py
```

**Review by DarkLight1337 — APPROVED** [review]

**Comment by dtrifiro** [comment]
How about trying something like this https://github.com/vllm-project/vllm/pull/4738?

**Pull request merged**